interview-questions.md

```diff
@@ -5,57 +5,95 @@ Answer the following questions. First, without external resources. Challenge you
 1. Why would you use the method super()?
 
   Your answer:
+  You use super to pass props from the contructor in the parent class. It is used when defining a class, to get information from the parent class. 
 
   Researched answer:
-
+  The super keyword is used to access and call functions on an object's parent. When used in a constructor, the super keyword appears alone and must be used before the this keyword is used. The super keyword can also be used to call functions on a parent object.
 
 
 2. What is a virtual DOM?
 
   Your answer:
+  The user interface. It is what the code looks like on the page. A document object model, creates UI, takes HTML and shows it on the page (ReactDOM). The visual representation of our code. we have to update the DOM every time we makde changes. The virtual DOM allows updates to happen in smooth and efficient way, allows React to make updates. 
 
   Researched answer:
+  A DOM is an interface used with Javascript by using JSON data.
 
+  In React, for every DOM object, there is a corresponding “virtual DOM object.” A virtual DOM object is a representation of a DOM object, like a lightweight copy. A virtual DOM object has the same properties as a real DOM object, but it lacks the real thing’s power to directly change what’s on the screen. Manipulating the DOM is slow. Manipulating the virtual DOM is much faster, because nothing gets drawn onscreen. Think of manipulating the virtual DOM as editing a blueprint, as opposed to moving rooms in an actual house.
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
   Your answer:
+  You will need an HTML tag, a head that includes React scripts, a body that includes a react container, and a script with the content you want. Inside the script, you will have the class which extends from React.Component, a render, return, the text you want ("Hello World"), the domContainer, and ReactDOM.render.
```

Comment:
Almost! Since we went over this in our review I'll keep this brief, but what you're describing is using react in an html document, but I think what is being asked here is about using something more along the lines of a functional, or perhaps a 'dumb'/stateless component.